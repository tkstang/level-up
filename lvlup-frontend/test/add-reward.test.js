import AddReward from '../src/components/Admin/rewards/add-reward//add-reward-main';
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import thunk from 'redux-thunk';
import configureMockStore from 'redux-mock-store';
import lvlupApp from '../src/reducers/index';

const middlewares = [thunk];
const mockStore = configureMockStore(middlewares);

test('AddReward should render', () => {
  const store = mockStore({ lvlupApp });
  const add = shallow(
    <AddReward store={store} />,
  );
  expect(shallowToJson(add)).toMatchSnapshot();
});
