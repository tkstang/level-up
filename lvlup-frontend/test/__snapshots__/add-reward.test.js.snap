exports[`test AddReward should render 1`] = `
<div
  className="add-reward">
  <Connect(lifecycle(AddRewardCompleted)) />
</div>
`;
